### PR TITLE
Update docs for always verbose behavior

### PR DESCRIPTION
**Pull request by 7373Lacym**
The docs now show that in CI environments the verbose reporter is used unless --tap is specified for #890

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -975,7 +975,7 @@ In contrast AVA is highly opinionated and runs tests concurrently, with a separa
 
 ### How can I use custom reporters?
 
-AVA supports the TAP format and thus is compatible with any [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters). Use the [`--tap` flag](#optional-tap-output) to enable TAP output.
+AVA supports the TAP format and thus is compatible with any [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters). Use the [`--tap` flag](#optional-tap-output) to enable TAP output. Please note, the verbose reporter is always used in CI environments unless [`--tap`](#optional-tap-output) is specified
```
Comment (jamestalmage) [inline readme.md]: `Please note, the verbose reporter` => `The verbose reporter`

**Comment by 7373Lacym** [comment]
Updated!  Thanks for the feedback!

**Comment by jamestalmage** [comment]
LGTM!

**Comment by sindresorhus** [comment]
Is this really the best place to put that information? People not wanting to use custom reporters might also want to know about the behavior. I would put it in the CLI section: https://github.com/avajs/ava#cli

**Comment by jamestalmage** [comment]
I almost left this same comment, but couldn't come up with a better place to put it.

Thinking about it more, It definitely shouldn't go here - if users are interested in custom reporters, then they definitely don't care about this behavior at all.

I think we should create a whole new "reporters" section that encompasses this.

- It should show screenshots from both the verbose and mini reporter, and discuss how to set up TAP reporters in more detail.

- We could explain this behavior there.

- We could eliminate this FAQ

- We could move the [clean stack traces section](https://github.com/avajs/ava#clean-stack-traces) in there.

**Comment by sindresorhus** [comment]
@jamestalmage Agreed

**Comment by jamestalmage** [comment]
@7373Lacym Are you able to update this PR as suggested above? No worries if the answer is no, someone else can take what you've done so far and run with it.

**Comment by 7373Lacym** [comment]
Sorry Guys,  internship started so I cannot get to this at the moment.  If it's still around in a few weeks I will pick it back up.

**Comment by MK-61** [comment]
Hello. Can I take this and continue, please?

**Comment by sindresorhus** [comment]
Closing in favor of #955.

**Pull request closed without merging**